go/api/go_api/session.py

```diff
@@ -12,7 +13,8 @@ class SessionStore(SessionBase):
     """
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
-        self.session_manager = vumi_api().session_manager
+        self.session_manager = SessionManager(
+            get_redis_manager().sub_manager('session_manager'))
```

Comment:
We can put them in class methods on `SessionManager` or something, I suppose. The real problem is that `VumiApi` and friends make all these things together and there's no easy way to get one without the others.